routes/checks/runs/list-check-runs-in-a-check-suite.json

```diff
@@ -19,7 +19,7 @@
       "location": "url"
     },
     {
-      "name": "id",
+      "name": "check",
```

Comment:
That should be `check_suit_id`. Problem is the dash in the URL parameter, I reported the issue to the docs team